scripts/main/album.js

```diff
@@ -1221,8 +1221,9 @@ album.delete = function (albumIDs) {
 		msg = `<p>` + lychee.locale["DELETE_UNSORTED_CONFIRM"] + `</p>`;
 	} else if (albumIDs.length === 1) {
 		let albumTitle = "";
+		const isTagAlbum = albums.isTagAlbum(albumIDs[0]);
 
-		action.title = lychee.locale["DELETE_ALBUM_QUESTION"];
+		action.title = lychee.locale[isTagAlbum ? "DELETE_TAG_ALBUM_QUESTION" : "DELETE_ALBUM_QUESTION"];
```

Comment:
This only applies to dingle albums. The `DELETE_ALBUMS_QUESTION` is used below for multi-select.

Reply:
You are both correct :smiley:. Multi-select is handled separately, and I chose not to deal with it, because the message could only be correct if one selects only tag albums or only regular albums, but not a mix of both. I decided that improving the most common case was "good enough"...